ci/prow/operator-upgrade

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
+# Get the latest bundle image published
+latest_published_bundle=$(curl -sSL "https://datagrepper.engineering.redhat.com/raw?topic=/topic/VirtualTopic.eng.ci.redhat-container-image.index.built&delta=2592000&contains=kernel-module-management-operator-bundle-container" | jq '.raw_messages[0].msg.index.index_image')
```

Comment:
If we are releasing for two streams, say `1.y` and `2.y`, would it be possible that this returns a `1.y` bundle?